_includes/navbar.html

```diff
@@ -0,0 +1,15 @@
+<!--- SPDX-License-Identifier: Apache-2.0 --->
+<nav id="header">
+  <span id="left-side">
+    <a href="/">Home</a>
+    <a href="/main/">ACLE</a>
+    <a href="/mve_intrinsics/">MVE</a>
+    <a href="/morello/">Morello</a>
+    <a href="/neon_intrinsics/">Neon</a>
+  </span>
+  <img id="logo" src="/assets/images/Arm-logo-reverse-white.svg" alt="Logo">
+  <span id="right-side">
+    {% if page.version %} Version: {{ page.version }} &nbsp;:::&nbsp; {% endif %}
+    {% if page.date-of-issue %} Date: {{ page.date-of-issue }} &nbsp;:::&nbsp; {% endif %}
```

Comment:
Is there a way to have Jekyll assert that `page.version` and `page.date-of-issue` are present? That would prevent the accidental publishing of specs without this information?

If such assert mechanism is possible, please remove the if statement and use the assertion.